docs/howto/operate/new-hub/aws.md

```diff
@@ -1,72 +1,76 @@
 # Add a new hub in a AWS kops-based cluster
 
-The idea behind this guide is showcase the process of setting up a 
-[kops](https://kops.sigs.k8s.io/getting_started/aws/)-based AWS
-cluster and manually deploy a new hub on top of it using our deployer tool.
-This is a preliminary but fully functional and manual process. Once
-[#381](https://github.com/2i2c-org/pilot-hubs/issues/381) is resolved, we should be able
-to automate the hub deployment process as we currently do with the GKE-based hubs.
+The idea behind this guide is to showcase the process of setting up a 
+[kops](https://kops.sigs.k8s.io/getting_started/aws/)-based AWS cluster and manually
+deploy a new hub on top of it using our deployer tool.
+This is a preliminary but fully functional and manual process. Eventually, we should be
+able to automate the hub deployment process as we currently do with the GKE-based hubs.
 
 ```{note}
-We will continue working toward a definitive one once we figured out some of the
-discussions outlined in [#431](https://github.com/2i2c-org/pilot-hubs/issues/431).
+We are currently deploying `kops` and `EKS` clusters. Some of the discussions about
+which tool we should bless are outlined in [#431](https://github.com/2i2c-org/pilot-hubs/issues/431).
 ```
 
-## Create an AWS kops-based cluster
+## Create an AWS kops-based cluster (optional)
 
-Follow the instructions in [](new-cluster:aws).
+Follow the instructions in [](new-cluster:aws) if you are not deploying against an
+existing cluster.
 
-## Deploy the new hub
+(new-hub:aws)=
+## Deploy the new AWS hub
 
-1. First, `cd` back to the root of the repository
+Follow the steps outlined in [](new-hub:deploy) with the following modifications:
 
-2. Generate a kubeconfig that will be used by the deployer.
-   To do so, run the following command: 
+1. Generate a new config file for your cluster if there is not an existing one.
```

Comment:
Why would there not be an existing config file for a cluster? Is this not part of the "set up an AWS cluster docs"?